WebApp/public/scripts/video-player.js

```diff
@@ -51,6 +51,8 @@ export class VideoPlayer {
       stream.getTracks().forEach(t => t.stop());
     }
 
+	console.log("Create new RTCPeerConnection")

```

Comment:
Do you need these changes?

It seems tab format is different.
Could you fix it if you need it?